interface-definitions/service_ntp.xml.in

```diff
@@ -13,6 +13,74 @@
           #include <include/generic-interface.xml.i>
           #include <include/listen-address.xml.i>
           #include <include/interface/vrf.xml.i>
+          <node name="ptp">
+            <properties>
+              <help>Enable Precision Time Protocol (PTP) transport</help>
+            </properties>
+            <children>
+              #include <include/port-number.xml.i>
+              <leafNode name="port">
+                <defaultValue>319</defaultValue>
+              </leafNode>
+              <node name="timestamp">
```

Comment:
Should this be nested under `ptp`?
chrony also supports timestamping of NTP packets (without the PTP transport) if the NIC supports it. This is the simplest setup when the system supports it and does not require using the ptp extensions or any non-standard NTP ports.